ui/app/styles/core/tag.scss

```diff
@@ -3,14 +3,22 @@
  * SPDX-License-Identifier: MPL-2.0
  */
 
+// This file defines the classes for .tag
+
 .tag:not(body) {
+  align-items: center;
   background-color: $ui-gray-100;
   border-radius: 2px;
   color: $grey;
+  display: inline-flex;
+  font-size: 0.75rem;
+  font-weight: normal;
   height: auto;
-  padding: 0 $size-10;
+  justify-content: center;
+  line-height: 1.5;
   margin-right: 0.5rem;
-  font-weight: normal;
+  padding: 0 $size-10;
+  white-space: nowrap;
```

Comment:
Yep. This is probably the hardest part of merging in bulma css: Figuring out which bulma css we used and where it fits in our old structure. So when you see new lines added like this into existing css styles, it's because bulma also styled this class selector (we were overriding it) but we still relied on these inserted styles.